welcome_wizard/views.py

```diff
@@ -48,140 +58,182 @@ def check_sync(instance, request):
         enqueue_pull_git_repository_and_refresh_data(repo, request.user)
 
 
-class ManufacturerListView(generic.ObjectListView):
-    """Table of all Manufacturers discovered in the Git Repository."""
+class ManufacturerImportUIViewSet(ObjectDetailViewMixin, ObjectListViewMixin, ObjectChangeLogViewMixin):  # pylint: disable=abstract-method
+    """List view for ManufacturerImport."""
 
     permission_required = "welcome_wizard.view_manufacturerimport"
-    table = ManufacturerImportTable
     queryset = ManufacturerImport.objects.all()
+    table_class = ManufacturerImportTable
+    filterset_class = ManufacturerImportFilterSet
+    filterset_form_class = ManufacturerImportFilterForm
     action_buttons = ()
-    template_name = "welcome_wizard/manufacturer.html"
-    filterset = ManufacturerImportFilterSet
-    filterset_form = ManufacturerImportFilterForm
-
-    def get(self, request, *args, **kwargs):
-        """Add Check Sync to Get."""
-        check_sync(instance=self, request=request)
-        return super().get(request, *args, **kwargs)
-
+    serializer_class = serializers.Serializer
```

Comment:
This should be using `welcome_wizard.api.serializers.ManufacturerImportSerializer`